### PR TITLE
add sharing buttons to market card

### LINKED ISSUE #4547: Add sharing to market cards
Design:
https://www.figma.com/file/Do4qnkCNM8P7lwBmCdYZCS4X/Market-Cards?node-id=3026%3A1210

Replace the 3 dots menu with the share button.

Clicking this share icon brings up an item list menu
Ignore all other styling that is in the item list design until finalised (grey colour etc)

Use same icons from market list page. Icons are 16px x 16px

### PR DESCRIPTION
**Pull request by andrewdamelio**
closes #4547 
Screen Shot 2020-01-10 at 12 41 01 PM

### CODE REVIEW

**Comment by bthaile** [comment]
Looks like the tracking events will get triggered for sharing, can you confirm?

**Comment by andrewdamelio** [comment]
@bthaile we currently track when they are clicked on the Market Page, i would assume we also want to the same events firing when they are clicked from the Market Card 

I will confirm

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). 🎉
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/0elhxon**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

**Comment by andrewdamelio** [comment]
> Looks like the tracking events will get triggered for sharing, can you confirm?

looks good

File: packages/augur-ui/src/modules/market/components/common/social-media-buttons.tsx [hunk 1]
```diff
@@ -31,22 +31,54 @@ export const SocialMediaButtons = (props: SocialMediaButtonsProps) => {
...
+
   return (
     <>
       <button
-        id="facebookButton"
+        id='facebookButton'
         onClick={() => {
-          props.sendFacebookShare(props.marketAddress, props.marketDescription);
-          showFacebookShare(encodedMarketUrl, encodedMarketDescription);
+          handleFacebookClick(props);
```
Comment (bthaile) [inline packages/augur-ui/src/modules/market/components/common/social-media-buttons.tsx]: I don't see a reason to move this logic around

**Review by bthaile — APPROVED** [review]
looks like the tracking is taken care of by container

**Pull request merged**